pkg/plugins/backendplugin/client.go

```diff
@@ -97,6 +101,16 @@ func NewRendererPluginDescriptor(pluginID, executablePath string, startFns Plugi
 			DefaultProtocolVersion: {
 				pluginID: &rendererV1.RendererPluginImpl{},
 			},
+			grpcplugin.ProtocolVersion: {
```

Comment:
I think we should extract this and the code at line 79 into a defaultStruct or function. I can totally see this being an hours-long debugging journey in the future.

Reply:
Good point. Extracted it to a function.